docs/embrace-api/supported_metrics_and_queries.md

```diff
@@ -4,14 +4,15 @@ description: Learn about the metrics and queries supported by the Embrace API
 sidebar_position: 3
 ---
 
-# Metrics API Supported Metrics
+# Supported Metrics
 
-## Standard Metrics
+The following metrics are supported. Metrics with the suffix "_total" are gauges.
```

Comment:
This is common across all the metrics, I moved it out of standard metrics section